### ISSUE TITLE
Issue 19.  As a user I want intuitive styling and AA compliancy

### ISSUE THREAD

**Issue by Bustamor** [the-collab-lab/tcl-54-smart-shopping-list#37]
## Summary

The appearance of all pages should be consistent and allow as much accessibility as possible.  Taking into consideration **font**, **colors**, **mobile** **devices** and user needs, we want the user to be able to interact comfortably with the app.  Contrast, color choices and AA compliancy are to be kept in mind when styling.  Screen size should not hinder user experience.  

## Acceptance Criteria

- [x] Font should be uniform across all pages and views
     - [x] Header - Lustria
     - [x] Paragraph - Lato

- [x] Colors
     - [x] Chosen with regards to AA compliancy
     - [x] Should be consistent across all pages
 
- [x] Buttons and links should be easy to use and understand
     - [x] Clear labels on buttons and links
     - [x] Aria labels on buttons
 
- [x] Accessible on a mobile device
     - [x] Resizing the screen doesn’t cause UI issues
     - [x] Ensure onClick actions still work (toasts, submitting forms, clearing filter, etc.)

- [x] Notifications should be easy to understand and act upon
     - [x] Consistent hot toast styling
     - [x] Clear next steps depending on what a notification box says
     
- [x] App is designed with all accessibility needs in mind
     - [x] Check app against accessibility add ons
     - [x] Make sure everything is screen-reader friendly
     
- [x] Update navbar

## Resources
https://chrome.google.com/webstore/detail/sas-accessibility-advisor/oajhmnajfdmljdknoeohidpjjdokokpl
